src/main.cpp

```diff
@@ -597,7 +594,7 @@ void GetGlobalStatus()
         GlobalStatusStruct.magnitude = RoundToString(boincmagnitude,2);
         GlobalStatusStruct.ETTS = RoundToString(dETTS,3);
         GlobalStatusStruct.ERRperday = RoundToString(boincmagnitude * GRCMagnitudeUnit(GetAdjustedTime()),2);
-        GlobalStatusStruct.cpid = GlobalCPUMiningCPID.cpid;
+        GlobalStatusStruct.cpid = NN::Researcher::Get()->Id().ToString();
```

Comment:
Is this the same as `NN::GetPrimaryCPID`?

Reply:
Yeah, and I probably should have used that here... `NN::GetPrimaryCpid()` does the string conversion...it's a helper for the transition to the `NN::Cpid` type, and I've been using it to help keep track of the places that still have a string.

Reply:
Switched!